Purchases.xcodeproj/project.pbxproj

```diff
@@ -7,24 +7,31 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
-		2D1A28CE24AA7224006BE931 /* LocalReceiptParser.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2D1A28CC24AA6F81006BE931 /* LocalReceiptParser.swift */; };
```

Comment:
changes here will be hard to read because xcode. 
summary: 
- created a new target for swift and a new test target for it
- move swift files into the main swift target and test swift target
- small cleanup of build settings here and there.